osmosis-1/osmosis-1.assetlist.json

```diff
@@ -6375,6 +6375,94 @@
         "osmosis-info",
         "OSMO:883"
       ]
+    },
+    {
```

Comment:
please remove any change to the osmosis-1.assetlist.json file. I see some minor errors, plus it will be auto-generated anyway based on the zone file addition.